code/modules/research/designs.dm

```diff
@@ -188,8 +188,8 @@
 /datum/design/item/powercell/contact
 	name = "contact energy"
 	id = "contact_energy"
-	req_tech = list(TECH_POWER = 3, TECH_MATERIAL = 2)
-	materials = list(MATERIAL_STEEL = 1250, MATERIAL_GLASS = 250, "uranium" = 125)
+	req_tech = list(TECH_POWER = 3, TECH_MATERIAL = 3)
+	materials = list(MATERIAL_STEEL = 2500, MATERIAL_PLASTEEL = 5000, "diamond" = 1300)
```

Comment:
is plasteel obtainable? requiring two rare materials seems like a huge leap in costs

Reply:
Plasteel is easily made, ordered or scavenged from the various maintenance hideyholes.